_sass/_components/_logo-bar.scss

```diff
@@ -101,11 +82,17 @@ $logo-bar-scale: 14px;
           max-height: $logo-bar-scale * 1.25;
         }
 
-        &[src*="zipkin_small.png"] {
+        &[src*="grpc"] {
```

Comment:
I am not enthusiastic about putting yaml-like content in here... is there an easy alternative to this approach?